esrally/reporter.py

```diff
@@ -34,6 +34,26 @@ def print_internal(message):
 def print_header(message):
     print_internal(console.format.bold(message))
 
+def write_single_report(self, report_file, headers, data, write_header=True, show_also_in_console=True):
+    report_format = self._config.opts("reporting", "format")
```

Comment:
You need to pass the report format as a parameter instead of using the config here.